### PR TITLE
Rudder publishing

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/faqs/yes-you-can-faq**](https://docs-qelksbu48.avo-dev.app/faqs/yes-you-can-faq) | [**/implementation/devs-101**](https://docs-qelksbu48.avo-dev.app/implementation/devs-101) |
|:-:|:-:|
|<a href="https://docs-qelksbu48.avo-dev.app/faqs/yes-you-can-faq">Screenshot of /faqs/yes-you-can-faq</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/04bOEa20g42fmyq8_DsSPVCj3noQISZwF_nD-mRbXEiuITEgf0MYhIOnFY2OhVGfkKAUjylAF9yYYivSlWz-gg.png">(view full size)</a> |<a href="https://docs-qelksbu48.avo-dev.app/implementation/devs-101">Screenshot of /implementation/devs-101</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/8uqRzeYhwtVZLLg2p6sE_-SnivEj98XwLvijJEFsXVVVvaE58u072LLu_hPzRbKtitigGktmcy_QyWLr1TZdLg.png">(view full size)</a> |



| [**…ation/reference/csharp**](https://docs-qelksbu48.avo-dev.app/implementation/reference/csharp) | [**…ntation/reference/java**](https://docs-qelksbu48.avo-dev.app/implementation/reference/java) |
|:-:|:-:|
|<a href="https://docs-qelksbu48.avo-dev.app/implementation/reference/csharp">Screenshot of /implementation/reference/csharp</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/sNglZhA2b8skXk8w0G_8BumuJ9sWNf4rcGmavER_o_IznS1HOQmK5o1pb3Hz89nR9oFDqpdokCrl2GCa4FANxw.png">(view full size)</a> |<a href="https://docs-qelksbu48.avo-dev.app/implementation/reference/java">Screenshot of /implementation/reference/java</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/_6WQCmSBuszVgi_ODdqx8RZAzoqumD1YL0vYRqMp3ngPYeC4bg0bdG-G4Uc21hnKMRQDq8trDreaupnICesWgg.png">(view full size)</a> |



| [**…n/reference/javascript**](https://docs-qelksbu48.avo-dev.app/implementation/reference/javascript) | [**…ation/reference/kotlin**](https://docs-qelksbu48.avo-dev.app/implementation/reference/kotlin) |
|:-:|:-:|
|<a href="https://docs-qelksbu48.avo-dev.app/implementation/reference/javascript">Screenshot of /implementation/reference/javascript</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Ml2JLg5yy4m-3pj4TDhur5ssqHT6fufxX0oNsE3f0B_9Wgw_jN-xGAFbGSrF2iKZQ4uZSEqte1tePLjSu26GZQ.png">(view full size)</a> |<a href="https://docs-qelksbu48.avo-dev.app/implementation/reference/kotlin">Screenshot of /implementation/reference/kotlin</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/TVCWvIMOUOoKHL2VCWrNtZfAHHGVpjLtjjikF5pwZ2fETg5vsKgpgqQtpT6fbVWYNe9WDwMts9LcNEsy0f8Vwg.png">(view full size)</a> |



And 11 other routes:
- [**/implementation/reference/objc**](https://docs-qelksbu48.avo-dev.app/implementation/reference/objc)
- [**/implementation/reference/php**](https://docs-qelksbu48.avo-dev.app/implementation/reference/php)
- [**/implementation/reference/python**](https://docs-qelksbu48.avo-dev.app/implementation/reference/python)
- [**/implementation/reference/reasonml**](https://docs-qelksbu48.avo-dev.app/implementation/reference/reasonml)
- [**/implementation/reference/rescript**](https://docs-qelksbu48.avo-dev.app/implementation/reference/rescript)
- [**/implementation/reference/ruby**](https://docs-qelksbu48.avo-dev.app/implementation/reference/ruby)
- [**/implementation/reference/swift**](https://docs-qelksbu48.avo-dev.app/implementation/reference/swift)
- [**/implementation/reference/typescript**](https://docs-qelksbu48.avo-dev.app/implementation/reference/typescript)
- [**/**](https://docs-qelksbu48.avo-dev.app/)
- [**/workspace/integrations**](https://docs-qelksbu48.avo-dev.app/workspace/integrations)
- [**/workspace/tracking-plan/publishing**](https://docs-qelksbu48.avo-dev.app/workspace/tracking-plan/publishing)



Commit d2db3a845f3220b2fe08d29bb855a53eb62b9b8a (https://docs-qelksbu48.avo-dev.app).

**Review by hugihlynsson — APPROVED** [review]
Nice! 👏

File: pages/workspace/integrations.mdx [hunk 1]
```diff
@@ -11,7 +11,8 @@ You can integrate Avo with your existing tools to streamline your analytics work
 <a href="#schema-management">Schema management</a>:
 
 - <a href="#webhook">Webhook</a>: Sync your analytics schema to any endpoint to manage downstream schemas such as ingestion time validation or SQL table structures  
-- <a href="#segment-protocols">Segment Protocols</a>: Sync your analytics schema to Segment Protocols  
+- <a href="#segment-protocols">Segment Protocols</a>: Sync your analytics schema to Segment Protocols
+- <a href="#rudderstack">RudderStack</a>: Sync your analytics schema to RudderStack
```
Comment (logason) [inline pages/workspace/integrations.mdx:15]:
```suggestion
- <a href="#rudderstack">RudderStack Tracking Plans</a>: Sync your analytics schema to RudderStack
```

File: pages/workspace/integrations.mdx [hunk 2]
```diff
@@ -36,10 +37,12 @@ With Avo you can _<Link passHref href="/workspace/tracking-plan/publishing">Publ
 ### Segment Protocols
 <Link passHref href="/workspace/tracking-plan/publishing#segment-protocols">Publish to Segment Protocols</Link> to sync your analytics schema downstream
 
+### RudderStack
```
Comment (logason) [inline pages/workspace/integrations.mdx:40]:
```suggestion
### RudderStack Tracking Plans
```

File: pages/workspace/tracking-plan/publishing.mdx [hunk 3]
```diff
@@ -150,6 +151,30 @@ On publish, Avo will merge your Avo tracking plan with any existing events in Le
 
 Avo will not impact any events that are only defined in Lexicon, or events that have not been added to the Lexicon integration in Avo.
 
+<img
+  src={require('../../../images/publishing_rudderstack.svg')}
+  alt="Amplitude"
+  style={{ margin: '32px 0 -32px 0' }}
+/>
+
+## <a name="rudderstack"></a> RudderStack
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:160]:
```suggestion
## <a name="rudderstack"></a> RudderStack Tracking Plans
```

File: pages/workspace/tracking-plan/publishing.mdx [hunk 4]
```diff
@@ -150,6 +151,30 @@ On publish, Avo will merge your Avo tracking plan with any existing events in Le
...
+
+The integration to [RudderStack](https://rudderstack.com/blog/dogfooding-at-rudderstack-tracking-plans-part-1/) allows for your Tracking Plan to be published into RudderStack whenever branches are merged in your workspace or with a push of a button.
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:162]: Hemingway adjustments:

The Rudderstack integration enables you to publish your Avo Tracking Plan into RudderStack. You can configure the integration to publish whenever a branch is merged or with a push of a button.

File: pages/workspace/tracking-plan/publishing.mdx [hunk 5]
```diff
@@ -150,6 +151,30 @@ On publish, Avo will merge your Avo tracking plan with any existing events in Le
...
+
+### Configure the RudderStack integration
+
+To enable the direct integration with Segment Protocols, you need to configure it with the following parameters from RudderStack.
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:166]:
```suggestion
To enable the direct integration with RudderStack Tracking Plans, you need to configure it with the following parameters from RudderStack.
```

File: pages/workspace/tracking-plan/publishing.mdx [hunk 6]
```diff
@@ -150,6 +151,30 @@ On publish, Avo will merge your Avo tracking plan with any existing events in Le
...
+
+- **RudderStack Account Email Address**: The email of a RudderStack account with Read-Write access
+- **RudderStack Tracking Plan Name**: The display name of the RudderStack Tracking Plan you want to publish to
+- **RudderStack Access Token**: [Create new Personal Access Token](https://rudderstack.com/docs/transformations/api-access-token/#access-token) for the account with the email address you provided above
+
+### Publishing to RudderStack
+
+Avo will override all events in the RudderStack Tracking Plan that are being imported from Avo. Any changes made to an event in RudderStack Tracking Plan between Avo publish will be overridden on next Avo publish.
+
+Avo will not override any events that are only defined in RudderStack, or events that have not been added to the RudderStack integration in Avo.
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:176]:
```suggestion
Avo will not override any events that are only defined in RudderStack, or events that have not been excluded from the RudderStack integration in Avo with filters.
```

**Review by logason — APPROVED** [review]
Few minor comments, otherwise LGTM 👍

**Pull request merged**